tensorflow-notebook/templates/service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "tensorflow-notebook.fullname" . }}
```

Comment:
I would create two different services to have two different endpoints. Does it make sense?

Reply:
why is that?

Reply:
There two containers running in the pod, the jupyter notebook and the tensorflow dashboard, I think both should have public endpoints

Reply:
Ah right. if both are meant to be used by the jupyter user it makes sense.

Reply:
I have added the ingress to control the route where the endpoint of the Jupiter notebook is exposed (the tensorboard does not admit it correctly).
Without the auto-generation of ingress I don't know if there is an advantage in having two services but functionally it is the same.